CONTRIBUTING.md

```diff
@@ -5,6 +5,12 @@ developers and one time contributors.  This guide should help you to create
 contributions that can be easily integrated into the FreeOrion project.
 
 
+## Respect the FreeOrion Vision Statement when contributing
+
+All contributions should follow the spirit of the [FreeOrion Vision Statement].
+We can't consider any contributions that miss or oppose the Vision Statement.
```

Comment:
Yeah, that section doesn't hold much value.  In previous commits I moved the vision statement into that file, because I didn't want to touch it first but the vision statement felt out of place here too so I added this section.